dev/bench/bench.sh

```diff
@@ -572,3 +572,10 @@ $not_installable_coq_opam_packages"
 fi
 
 zulip_edit "Bench complete: all packages successfully installed."
+
+## Generate line summary
```

Comment:
Should be done before the `if [ -n "$not_installable_coq_opam_packages" ]; then`